geometry_msgs/PoseStamped pose_start
sensor_msgs/JointState joint_state_start
sensor_msgs/JointState joint_state_end

---

geometry_msgs/PoseStamped pose_end
